### PR TITLE
Add flag to only search IRIS (avoiding Winston!)

### PR DESCRIPTION
**Pull request by liamtoney**
This addresses https://github.com/uafgeotools/waveform_collection/issues/18#issuecomment-1057636587.

### CODE REVIEW

**Comment by liamtoney** [comment]
Note that the diff looks large; this is only because of indenting changes!

**Comment by liamtoney** [comment]
There was some discussion of having the flag set to `True` by default — as we usually only seek IRIS data.

File: waveform_collection/server.py [hunk 1]
```diff
@@ -275,7 +275,7 @@ def gather_waveforms_bulk(lon_0, lat_0, max_radius, starttime, endtime,
                           channel, network='*', station='*', location='*',
                           time_buffer=0, merge_fill_value=0, trim_fill_value=0,
                           remove_response=False, watc_url=None,
-                          watc_username=None, watc_password=None):
+                          watc_username=None, watc_password=None, iris_only=False):
```
Comment (davidfee5) [inline waveform_collection/server.py:278]: As discussed, I think we should set default to iris_only=True since that is the standard FDSN server we use

**Review by davidfee5 — CHANGES_REQUESTED** [review]
Looks good other than the iris_only default change!

**Comment by liamtoney** [comment]
Because of this change, any code w/o this flag will only be grabbing IRIS data. This mightttt break something, but I don't think it is crucial enough. So we can merge.

**Pull request merged**